apps/web/modules/ui/components/editor/components/toolbar-plugin.tsx

```
@@ -461,93 +463,98 @@ export const ToolbarPlugin = (props: TextEditorProps & { container: HTMLElement
-                variant="ghost"
-                type="button"
-                onClick={insertLink}
-                className={isLink ? "bg-subtle active-button" : "inactive-button"}>
-                <Link />
-              </Button>
-              {isLink ? (
-                createPortal(<FloatingLinkEditor editor={editor} />, props.container ?? document.body)
-              ) : (
-                <></>
-              )}
+              <span className={"icon" + blockType} />
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)